<a href="https://colab.research.google.com/github/syuan611/115-1/blob/main/411403930_H2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [5]:
x=int(input("請輸入目前金幣數量: "))
y=int(input("請輸入獲得金幣數量: "))
z=(x+y)
print(f"任務完成！你現在有{z}枚金幣")

請輸入目前金幣數量: 500
請輸入獲得金幣數量: 250
任務完成！你現在有750枚金幣


題目：請寫一個 Python 程式，讓使用者輸入目前金幣數量和完成任務獲得的金幣，計算任務完成後的總金幣，並輸出結果（利用格式化輸出方式）。



輸入範例：
請輸入目前金幣數量:500

請輸入獲得金幣數量:250



輸出範例：
任務完成！你現在有750枚金幣